### PR TITLE
Clarifying docs around process_template

### PR DESCRIPTION
**Pull request by salilgupta1**
Noting that K8s doesn't support templates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ client = Kubeclient::Client.new 'https://localhost:8443/api/' , 'v1',
 
 You can find information about token in [this guide](http://kubernetes.io/docs/user-guide/accessing-the-cluster/) and in [this reference](http://kubernetes.io/docs/admin/authentication/).
 
-You can also use kubeclient with non-blocking sockets such as Celluloid::IO, see [here](https://github.com/httprb/http/wiki/Parallel-requests-with-Celluloid%3A%3AIO) 
+You can also use kubeclient with non-blocking sockets such as Celluloid::IO, see [here](https://github.com/httprb/http/wiki/Parallel-requests-with-Celluloid%3A%3AIO)
```
Comment (simon3z) [inline README.md]: @salilgupta1 this change is unrelated to process_template, can you remove it from the PR? Thanks.

**Comment by salilgupta1** [comment]
@simon3z squashed the commits
